www/video/render.inc.php

```diff
@@ -757,26 +743,19 @@ function DrawTest(&$test, $renderInfo, $frameTime, $im)
         $path = $test['frames'][$frame_ms]['path'];
     }
 
-    $need_grey = false;
```

Comment:
I'm a bit confused about the need_grey/applyGrey removal.

It looks like $applyGray defaults to true unless a query param explicitly disables it. The logic below determines if a given frame is the last frame and is past the end time of the video and if so, grays out the frame.

It is the logic for ending each side of a comparison video with a gray page content after the page is finished and doesn't appear to be unused.

Reply:
so `$applyGrey` was defined in one function, used in another, so it wasn't doing anything which lead me to delete a bunch of things. Now I moved it to where it's used and the feature is back on (currently broken in www, see https://www.webpagetest.org/video/view.php?tests=221010_AiDcVV_EXQ-r:1-c:0)